server/routes/board.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  // show kanban board
```

Comment:
You shouldn't have an empty route. If anything just delete this till you're ready to work on it.